### DISCUSSION TITLE
Error http 500 after a manual modification of order's state by the back office

### DISCUSSION THREAD

**Discussion by kevintdc** [PrestaShop/PrestaShop#35409] [Q&A]
Hello, i have a http 500 error when i change the order's state manually. 

I'm on 1.7.8.7 prestashop and 7.4 php. And host by OVH

Why i do this ? 

Because i have a customer who pay monthly and i dont have any paiement options for this. So i let him choose the transfert and change the order's state "Waiting for the transfert" by "In preparation" manually. Here is the thing. If i do this manually, you go on the site and click on an article, you have the http 500 error. If i change the state by the origin state for the transfert, the error disapear.

I ask Prestashop support but they cant help me to solve this one. If some one can give me some tips ! 

Thank you
![Capture d'écran 2024-02-05 092126](https://github.com/PrestaShop/PrestaShop/assets/101325383/e90fea03-a99d-47b3-9f62-0163d8f42757)
![Capture d'écran 2024-02-05 092158](https://github.com/PrestaShop/PrestaShop/assets/101325383/e83b9ead-e01d-407c-9d1e-54d806648806)
![Capture d'écran 2024-02-05 092230](https://github.com/PrestaShop/PrestaShop/assets/101325383/219a3c86-85ed-4f64-b9a5-6ed2cff6e29d)
![Capture d'écran 2024-02-14 095252](https://github.com/PrestaShop/PrestaShop/assets/101325383/79657e3c-a682-41ac-98d3-f8ea47deede9)

**Comment by kpodemski** [comment]
Hello @kevintdc 

Open the file mentioned in the error: src/Adapter/Debug/DebugMode.php and change `opcache_invalidate($filename);` to `@opcache_invalidate($filename);`

**Reply by kevintdc** [reply]
Hello,

I did this but nothing change...
